src/Core/Grid/Position/UpdateHandler/PositionUpdateHandlerInterface.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * 2007-2018 PrestaShop.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Grid\Position\UpdateHandler;
+
+use PrestaShop\PrestaShop\Core\Grid\Position\Exception\PositionUpdateException;
+use PrestaShop\PrestaShop\Core\Grid\Position\PositionDefinitionInterface;
+
+/**
+ * Interface PositionUpdateHandlerInterface is used by GridPositionUpdater to
+ * manipulate the data, handling the manipulation in this interface allows the
+ * GridPositionUpdater to adapt more easily to different databases or any other
+ * persistence solutions.
+ */
+interface PositionUpdateHandlerInterface
+{
+    /**
```

Comment:
to complete my previous opinion about doc comments, I think it's totally ok to add a lot of information in interfaces! Am I an annoying reviewer? Probably :)